srtp/srtp.c

```diff
@@ -193,56 +214,74 @@ srtp_stream_alloc(srtp_stream_ctx_t **str_ptr,
   memset(str, 0, sizeof(srtp_stream_ctx_t));
   *str_ptr = str;
 
-  /* allocate cipher */
-  stat = srtp_crypto_kernel_alloc_cipher(p->rtp.cipher_type, 
-				    &str->rtp_cipher, 
-				    p->rtp.cipher_key_len,
-				    p->rtp.auth_tag_len); 
-  if (stat) {
-    srtp_stream_free(str);
-    return stat;
+  /* To keep backwards API compatible if someone is using multiple master
+   * keys then key should be set to NULL
```

Comment:
This is referring to the Policy key passed into SRTP.  The end user of SRPT if he/she is using MKI then they must set the Policy Key to NULL.